### DISCUSSION TITLE
Bi-directional charging

### DISCUSSION THREAD

**Discussion by nickvanbaelen** [SmartEVSE/SmartEVSE-2#58] [Q&A] [answered]
I heard that some vehicles are able to act as a "home battery" by pushing through rather than pulling power from the charging station. Does the Smart EVSE product allow for this type of bi-directional charging?

**Comment by mstegen** [comment, marked as answer]
No the SmartEVSE-2 or -3 does not speak the protocol to allow V2G.
You will need a compatible EV for that, and a EVSE that has a modem built-in, and speaks ISO15118 with the car. 

Some EV's have a V2L (vehicle to load) function, so that you can pull power directly from the EV, but this is usually limited to 3600W , and you can not connect this output to the grid.

**Comment by kirkholt** [comment]
V2L would be at VERY neat feature even when it is limited to 3600W.
The circuit required is quite simple as per this [video](https://www.youtube.com/watch?v=K22CFmSA1bM&t=746s)
